Orlando Walking Tours/Controllers/AddTourVC.swift

```diff
@@ -24,6 +23,7 @@ class AddTourVC: UIViewController, UITextFieldDelegate
 
     let notificationCenter = NSNotificationCenter.defaultCenter()
     var uuid: NSUUID?
+    var modelService: ModelService!
```

Comment:
either do what you did with Notification Center and initialise inline or change the ! to ?, because things could explode.